subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyMetadataRulesIntegrationTest.groovy

```diff
@@ -213,7 +213,7 @@ abstract class DependencyMetadataRulesIntegrationTest extends AbstractHttpDepend
                         withVariant("$variantToTest") { 
                             withDependencies {
                                 it.each {
-                                    it.version = '1.0'
+                                    it.version { prefer = '1.0' }
```

Comment:
I am surprised this works. `prefer '1.0'` should pass, but `setPrefer('1.0')`?

Reply:
Good catch. This must be groovy magic. The _private_ field in `DefaultMutableVersionConstraint` is called `prefer`.

Reply:
Ah yes that must be it. I guess we need to change the test to use `prefer '1.0'` for idiomatic Gradle.

Reply:
I thought I had changed it... I think I had two occurrences. Should be fixed now.